button-2/blocks.css

```diff
@@ -20,7 +20,10 @@ Description: Used to style Gutenberg blocks
 
 /* Captions */
 
-[class^="wp-block-"] figcaption {}
+[class^="wp-block-"] figcaption {
```

Comment:
Let's go ahead and copy [the original caption styles](https://github.com/Automattic/themes/blob/master/button-2/style.css#L1916) for the theme here. And then update [the editor styles](https://github.com/Automattic/themes/blob/master/button-2/editor-blocks.css#L156) to reflect this.


```
font-family: Lora, Garamond, serif;
font-style: italic;
text-align: center;
```

Reply:
I've addressed the requested changes in https://github.com/Automattic/themes/pull/1726/commits/d82c13be435af125f9140355f60e3aad73925170